### PR TITLE
Fix MSVC compiler warnings

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
reg.: declaration hides class member (C4458)

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
What the heck is wrong with "CI_Without Qt"?

**Comment by Jojo-Schmitz** [comment]
Besides the GitHub CI error there's another issue:

```
Severity	Code	Description	Project	File	Line	Suppression State	Details
Warning	MSB8027	Two or more files with the name of ioc.cpp will produce outputs to the same location. This can lead to an incorrect build result.  The files involved are ...\src\framework\global\thirdparty\kors_modularity\modularity\ioc.cpp, ...\src\framework\global\modularity\ioc.cpp.	muse_global	C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.CppBuild.targets	1119		
```

File: buildscripts/ci/withoutqt/setup.sh [hunk 1]
```diff
@@ -108,7 +108,7 @@ sudo apt-get install -y --no-install-recommends \
 
 # COMPILER
 
-gcc_version="11"
+gcc_version="10"
```
Comment (Jojo-Schmitz) [inline buildscripts/ci/withoutqt/setup.sh:111]: the regular (Linux) build(s) use GCC 10 too

File: src/playback/internal/playbackcontroller.cpp [hunk 2]
```diff
@@ -1467,14 +1467,14 @@ msecs_t PlaybackController::beatToMilliseconds(int measureIndex, int beatIndex)
...
-    return playback ? playback->tempoMultiplier() : 1.0;
+    INotationPlaybackPtr pb = notationPlayback();
+    return pb ? pb->tempoMultiplier() : 1.0;
 }
 
 void PlaybackController::setTempoMultiplier(double multiplier)
 {
-    INotationPlaybackPtr playback = notationPlayback();
-    if (!playback) {
+    INotationPlaybackPtr pb = notationPlayback();
```
[the remark this answers is not in the document]
Comment (Jojo-Schmitz) [reply src/playback/internal/playbackcontroller.cpp:1476]: as is the score of that `ab` furter up

[the remark this answers is not in the document]
Comment (Eism) [reply src/playback/internal/playbackcontroller.cpp:1476]: > And this method is rather small

This is temporary :)

[the remark this answers is not in the document]
Comment (Jojo-Schmitz) [reply src/playback/internal/playbackcontroller.cpp:1476]: Better name? `playbackPtr`, `notationPlayback`?

[the remark this answers is not in the document]
Comment (Jojo-Schmitz) [reply src/playback/internal/playbackcontroller.cpp]: Check now, whether you like it better...

**Pull request merged**